graph-db/src/utils/cmr/clearScrollSession.js

```diff
@@ -0,0 +1,19 @@
+const fetch = require('node-fetch');
+
+/**
+ * Given a CMR Scroll Session ID, clear that session from
```

Comment:
Sometimes you add the method name at the beginning of the comment block, sometimes you don't. Don't necessarily have a preference here but you should be consistent.